docs/src/content/docs/basic-concepts.mdx

```diff
@@ -86,10 +86,10 @@ Unlike React's concurrent rendering model, GenSX evaluates your workflow as a de
 
 GenSX provides two main types of components:
 
-1. **Components** (`gsx.Component`) - For standard synchronous or asynchronous operations
-2. **Streaming Components** (`gsx.StreamingComponent`) - For handling streaming responses from LLMs
+1. **Components** (`gsx.Component`) - Components that produce values and can handle both synchronous and asynchronous operations
```

Comment:
nit: `that produce an output value` -- still a singular output whether it be an array, object, string, number, etc.

Reply:
fair, updated